components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/FirefoxAccount.kt

```diff
@@ -140,7 +141,8 @@ class FirefoxAccount internal constructor(
         accessType: AccessType
     ) = scope.async {
         handleFxaExceptions(logger, "authorizeOAuthCode", { null }) {
-            inner.authorizeOAuthCode(clientId, scopes, state, accessType.msg)
+            val authParams = AuthorizationParams(clientId, scopes, state, accessType.msg)
+            inner.authorizeOAuthCode(authParams)
```

Comment:
Not necessarily for this PR, but flagging for future consideration - we should consider pushing the `AuthorizationParams` abstraction up through the API layers here, so that callers of the higher-level API can pass all the new properties in a sensible way.

Reply:
Agreed, we can do that in a follow-up.